src/bentoml/_internal/io_descriptors/json.py

```diff
@@ -128,8 +147,57 @@ def classify(input_data: IrisFeatures) -> NDArray[Any]:
            input_df = pd.DataFrame([input_data.dict(exclude={"request_id"})])
            return iris_clf_runner.run(input_df)
 
+    :obj:`JSON` also supports ``typing.TypedDict`` in addition to ``pydantic.BaseModel`` as a validation schema. Note that for Python < 3.9, you can add the following code block to your code in order to use ``TypedDict``:
+
+    .. code-block:: python
+
+        import sys
+
+        if sys.version_info[:2] >= 3.8
```

Comment:
```suggestion
        if sys.version_info[:2] >= (3, 8)
```

Sorry my bad on this